src/packages/block/block/workspace/block-workspace.context.ts

```diff
@@ -218,6 +229,11 @@ export class UmbBlockWorkspaceContext<LayoutDataType extends UmbBlockLayoutBaseM
 			throw new Error('Block Entries could not create block');
 		}
 
+		const blockType = this.#blockManager!.getBlockTypeOf(contentElementTypeId);
+		if (blockType?.label) {
+			this.setName(blockType.label);
```

Comment:
We are still missing the Label interpretation feature, meaning using the Label (template) would not necessary be very pretty.

How was that i v.13 did we show the interpolated label in the header of the modal? :-)

Reply:
Ah, you're right! The modal label shows the interpolated version of it. 🤦‍♂️ Shall I draft this until that gets resolved?

Reply:
Well, we need the label soon.. so maybe you could leave those parts commented out and instead have the name set to the Element Type Name?